### ISSUE TITLE
Gap in testing with the pretrained model

### ISSUE THREAD

**Issue by chongruo** [facebookresearch/SlowFast#35]
Similar to issue #28, I also have a gap in test results with the pretrained model. It is evaluated with SLOWFAST_4x16_R50 model.  The top-1 accuracy is only 74.37 (smaller than reported result 75.6). 

Here is the running script I used
![image](https://user-images.githubusercontent.com/6445892/69120626-e68dc900-0a4e-11ea-8502-0ce2faaac2f9.png)
The version of pyav on my size is 6.2.0. All videos are resized to the short edge size of 256.

There are roughly around 19,600 valid videos in my val dataset. For the missing videos, some links to videos are broken, and some videos only contain the audio track.

**Comment by takatosp1** [comment]
Hi,
Thanks for using the codebase. As offline discussed with #28, we found the difference is because of the validation data. I’ll prepare and share you the validation data with you offline. You could also download the validation from https://github.com/facebookresearch/video-nonlocal-net, which is the source of data we were using.

**Comment by takatosp1** [comment]
Update from the offline discussion: chongruo reported the number has matched after using the updated validation data.

**Comment by chongruo** [comment]
The val performance of the pretrained model is improved on a better dataset (#42). It is close to the reported number (within 0.41 gap for SlowFast_R50_4x16). As mentioned by @takatosp1, this reasonable gap is probably due to the model conversion from caffe2 to pytorch, or inaccurate decoding of some videos. Also, this dataset is incomplete (https://github.com/facebookresearch/video-nonlocal-net/blob/master/DATASET.md). We will keep looking into it for potential improvements. 

Thanks Haoqi for his help !
